cmd/romana.k8s

```diff
@@ -0,0 +1,403 @@
+#!/bin/bash
+
+# Copyright (c) 2016 Pani Networks
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# Usage subcommand.
+usage() {
+	echo "romana: command-line tool for Romana administration"
+	echo "Usage: romana <subcommmand> [options]"
+	echo "       romana <subcommmand> help"
+	echo "       romana [help]"
+	echo "Subcommands:"
+	echo "    add-host"
+	echo "    remove-host"
+	echo "    show-host"
+	echo "    create-owner"
+	echo "    delete-owner"
+	echo "    show-owner"
+	echo "    add-tier"
+	echo "    remove-tier"
+	echo "    add-policy"
+	echo "    remove-policy"
+	echo "    show-policy"
+}
+
+# Variables used within the script
+tenant_url=http://{{ stack_nodes.Controller.mgmt_ip }}:9602/
+tenant_path="tenants"
+topology_url=http://{{ stack_nodes.Controller.mgmt_ip }}:9603/
+topology_path="hosts"
+kube_apiserver_url=http://{{ stack_nodes.Controller.mgmt_ip }}:8080/
+romana_policy_path="apis/romana.io/demo/v1/namespaces/default/networkpolicys"
```

Comment:
This tool is just a placeholder until we have the real CLI tools (which probably won't be written in bash, and have a bit more validation).

The policy parts were just to support the demo; other parts support the installation as well, where it configures owners/tiers in the DBs we have.

In general, I think that it does belong in the romana CLI tool (proper one, once written), instead of expecting the user to get the curl commands just right, or having two tools -- a 'romana' one for internal bits, and a romana-k8s for k8s-specific bits.